### PR TITLE
Add Best in Europe 2019 logo to narrative

### PR DESCRIPTION
**Pull request by mdkalish**
viewport width ≤ 720 px
Screenshot 2019-05-10 at 13 44 22

viewport width > 720 px
Screenshot 2019-05-10 at 13 44 28

### CODE REVIEW

**Comment by vacas5** [comment]
can we get a conventional commit message on this? https://github.com/lonelyplanet/rizzo-next/blob/master/CONTRIBUTING.md

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "version:minor": "npm run release -- --release-as minor",
     "version:major": "npm run release -- --release-as major"
   },
-  "version": "0.38.2",
+  "version": "0.38.3",
```
Comment (vacas5) [inline package.json]: was there an attempt to version that failed? if so, can we rebase out the commit message?

**Comment by mdkalish** [comment]
> can we get a conventional commit message on this? https://github.com/lonelyplanet/rizzo-next/blob/master/CONTRIBUTING.md

Done. It would be great to have it in DFE too!

**Comment by vacas5** [comment]
> > can we get a conventional commit message on this? https://github.com/lonelyplanet/rizzo-next/blob/master/CONTRIBUTING.md
> 
> Done. It would be great to have it in DFE too!

it would be nice for sure. Feel free to add it using this as the example and we can fine tune it in a PR

**Pull request merged**